src/utils/validations/component/validations/with-unique-name.hoc.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import guid from '../../../helpers/guid/guid';
+
+const withUniqueName = WrappedComponent => props => (
+  <WrappedComponent name={ guid() } { ...props } />
```

Comment:
is this not going to continuously generate a new guid each render?

Reply:
The tests seem to indicate that the component's guid is persisted when I call update on the wrapper